src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -742,11 +748,26 @@ class RelayToTIRVisitor : public MixedModeMutator {
                                                GetRef<Function>(func));
         }
 
+        // Drop out the redundant arguments and their arg_types from the call to the global
+        // partitioned function.
```

Comment:
Drop out the redundant arguments, and the arg_types from the global partitioned function call

Reply:
ACK